src/main/java/org/jboss/tools/intellij/openshift/ui/sandbox/SandboxWorkflowPage.java

```diff
@@ -183,7 +183,7 @@ private void setStatus(String message) {
     }
 
     private void launchJob() {
-        ProgressManager.getInstance().run(new Task.Backgroundable(project, "Login to Red Hat Developer Sandbox", true) {
+        ProgressManager.getInstance().run(new Task.Backgroundable(project, "Login to red hat developer sandbox", true) {
```

Comment:
why suddenly remove Capitals letters ?

Reply:
Shouldn't be included in this PR since it's corrected in #739 (same issue discussed here: https://github.com/redhat-developer/intellij-openshift-connector/pull/739#pullrequestreview-1936050135). Reverting this change.